### DISCUSSION TITLE
vitepress为什么不推荐使用？有什么弊端吗

### DISCUSSION THREAD

**Discussion by nabaonan** [shengxinjing/it-roadmap#21] [提问]
![image](https://user-images.githubusercontent.com/17247526/144989922-bc208e22-dc3b-4e1e-9715-bec663e1982c.png)

**Comment by shengxinjing** [comment]
这个项目最早就是vitepress搭建的，后来发现迟迟不发布1.0，并且vuepress2已经支持vite了，所以就选择了vupress2+ vite构建的方式
